core/src/main/java/org/gbif/pipelines/core/interpretation/LocationInterpreter.java

```diff
@@ -212,17 +228,27 @@ static LocationInterpreter interpretCoordinateUncertaintyInMeters(Location locat
    * {@link DwcTerm#coordinatePrecision} interpretation.
    */
   static LocationInterpreter interpretCoordinatePrecision(Location locationRecord) {
-    return (ExtendedRecord extendedRecord) ->
-      SimpleTypeParser.parseDouble(extendedRecord, DwcTerm.coordinatePrecision, parseResult -> {
-        Interpretation<ExtendedRecord> interpretation = Interpretation.of(extendedRecord);
-        Double result = parseResult.orElse(null);
-        if (result != null && result >= COORDINATE_PRECISION_LOWER_BOUND && result <= COORDINATE_PRECISION_UPPER_BOUND) {
-          locationRecord.setCoordinatePrecision(result);
-        } else {
-          interpretation.withValidation(Trace.of(DwcTerm.coordinatePrecision.name(), IssueType.COORDINATE_PRECISION_INVALID));
-        }
-        return interpretation;
-      });
+    return (ExtendedRecord extendedRecord) -> SimpleTypeParser.parseDouble(extendedRecord,
+                                                                           DwcTerm.coordinatePrecision,
```

Comment:
indentation is different to others lambda functions

Reply:
fixed. That different indentation was done with our common formatter. If it's wrong I think we should fix it in order to be able to apply it everywhere.